tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
 ﻿<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="Mono.Android.NET_Tests">
-	<uses-sdk android:minSdkVersion="21" />
```

Comment:
Is this removal intentional?

Reply:
Yep, IIRC we generate that line automatically now, based on the csproj settings (or defaults)